notebooks/compare_esm_ssp_opt.ipynb

```diff
@@ -0,0 +1,691 @@
+{
```

Comment:
<p>I'm curious if you have returned to the catalog recently and if so, if it is more complete now?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NCAR/ctsm_py/pull/22/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>

Reply:
<p>I haven't been using the catalog much recently, but these commands still work for the plotting I wanted to do in this notebook. I did a test search to see if there were any additional models that had <code><em>experiment_id='esm-ssp585'</em></code> and <code>variable_id='npb'</code> and there aren't any new ones.</p>

Reply:
> I did a test search to see if there were any additional models that had experiment_id='esm-ssp585' and variable_id='nbp' and there aren't any new ones.

I updated the catalog today, and it appears that **nothing** has changed for `experiment_id='esm-ssp585'` and `variable_id='nbp'`

```python
In [5]: import intake                                                                               

In [6]: col = intake.open_esm_datastore("/glade/collections/cmip/catalog/intake-esm-datastore/catalo
   ...: gs/glade-cmip6.json")                                                                       

In [7]: col_subset = col.search(experiment_id='esm-ssp585')                                         

In [8]: col_subset = col.search(experiment_id='esm-ssp585', variable_id='nbp')                      

In [9]: col_subset                                                                                  
Out[9]: 
glade-cmip6-ESM Collection with 5 entries:
        > 1 activity_id(s)

        > 2 institution_id(s)

        > 2 source_id(s)

        > 1 experiment_id(s)

        > 3 member_id(s)

        > 1 table_id(s)

        > 1 variable_id(s)

        > 1 grid_label(s)

        > 0 dcpp_init_year(s)

        > 2 version(s)

        > 3 time_range(s)

        > 5 path(s)

In [10]: col_subset.df                                                                              
Out[10]: 
  activity_id institution_id  ...     time_range                                               path
0       C4MIP           NCAR  ...  201501-206412  /glade/collections/cmip/CMIP6/C4MIP/NCAR/CESM2...
1       C4MIP           NCAR  ...  206501-210012  /glade/collections/cmip/CMIP6/C4MIP/NCAR/CESM2...
2       C4MIP          CCCma  ...  201501-210012  /glade/collections/cmip/CMIP6/C4MIP/CCCma/CanE...
3       C4MIP          CCCma  ...  201501-210012  /glade/collections/cmip/CMIP6/C4MIP/CCCma/CanE...
4       C4MIP          CCCma  ...  201501-210012  /glade/collections/cmip/CMIP6/C4MIP/CCCma/CanE...

[5 rows x 12 columns]

In [11]: col_subset.unique(columns=['source_id'])                                                                                                    
Out[12]: {'source_id': {'count': 2, 'values': ['CESM2', 'CanESM5']}}
```